packages/docusaurus-theme-classic/src/theme/NavbarItem/DocNavbarItem.tsx

```diff
@@ -56,7 +56,7 @@ export default function DocNavbarItem({
       {...props}
       className={clsx(props.className, {
         [activeDocInfimaClassName]:
-          activeDoc && activeDoc.sidebar === doc.sidebar,
+          activeDoc?.sidebar && activeDoc.sidebar === doc.sidebar,
```

Comment:
in this case navbar items without sidebar always will be inactive?

Reply:
No it would work fine because it's already the default behavior for a link to be active when we are on the link target page. We only need this additional class to enable the "tab-like" behavior.